### DISCUSSION TITLE
How to make my graph normal?

### DISCUSSION THREAD

**Discussion by Rachmaninofff3** [NightscoutFoundation/xDrip#2308] [Q&A] [answered]
![Screenshot_20220905-013655_xDrip+](https://user-images.githubusercontent.com/112820132/188324181-2b235110-13c4-454a-9657-4bf2adcdb057.jpg)
Hi, sorry for my bad English before asking you a question. I have never seen this kind of big number on my x-drip graph. It just happened once I got a very high blood sugar figure right after I took off my libre sensor. Do you know how to make my graph normally? I meant.. I just wanna see the numbers from 0 to 300 or 250.. please share your ideas if you know how to do it. Thank you.

**Comment by Navid200** [comment, marked as answer]
Hi,

Your graph will go back to normal 24 hours after the high reading.

There is an open issue related to this: https://github.com/NightscoutFoundation/xDrip/issues/2173
I intend to propose a change to xDrip to improve user experience related to this.  However, I have not got to it yet.

**Reply by Rachmaninofff3** [reply]
Thank you so much for your answer. :)
